docs/platform/concepts/service-power-cycle.rst

```diff
@@ -0,0 +1,39 @@
+Service power cycle
```

Comment:
Is this "informative enough"? or should we talk about "Power OFF Power ON cycle"?

Reply:
Power cycle means power off then power on as many docs on line explain, e.g. https://techterms.com/definition/powercycle.